chia/plotting/plot_info.py

```diff
@@ -0,0 +1,21 @@
+import logging,sys
+
+from chia.plotting.plot_tools import load_plot
+
+log = logging.getLogger(__name__)
+
+
+def plot_info(file: str):
+    (plot_info,error) = load_plot(file)
+    if error!="":
+        print(error)
+        sys.exit(1)
+    else:
+        pr = plot_info.prover
+        id = pr.get_id()
+        print(f"path: {file}")
+        print(f"k: {pr.get_size()}")
+        print(f"id: {id.hex()}")
+        print(f"farmer public key: {plot_info.plot_public_key}")
+        print(f"pool public key: {plot_info.pool_public_key}")
+        sys.exit(0)
```

Comment:
I think you might get a test warnings also due to formatting issues such as blanks after commas, as in `a, b = f()`.

Btw. you can use some idioms to turn this into
```python
def plot_info(file: str):
    plot_info, error = load_plot(file)

    if error:
        print(error)
        sys.exit(1)
    else:
        print("\n".join([
            f"path: {file}", 
            f"k: {plot_info.prover.get_size()}",
            f"id: {pr.get_id().hex()}",
            f"farmer public key: {plot_info.plot_public_key}",
            f"pool public key: {plot_info.pool_public_key}",
        ])
        sys.exit(0)
```